.github/workflows/build_and_test.yml

```
@@ -26,3 +24,27 @@ jobs:
       run: ./gradlew jar
     - name: Test
       run: ./gradlew test
+      
+  build-and-push-anchor-platform:
+    runs-on: ubuntu-latest
```

❗ Jobs will run in parallel unless you explicitly link their dependencies. You'll need to add a `needs` to ensure this runs after the `build_and_test`.
```suggestion
  build-and-push-anchor-platform:
    needs: [build_and_test]
    runs-on: ubuntu-latest
```